src/main/java/net/sf/jabref/gui/importer/fetcher/EntryFetcher.java

```diff
@@ -8,6 +8,7 @@
 import net.sf.jabref.logic.importer.OutputPrinter;
 
 /**
+ * <b>use SearchBasedFetcher instead</b> <br>
```

Comment:
Do we really need to write proper javadoc documentation for deprecated fields or methods? I would rather invest my time in removing the old code. For me the previous syntax with method annotation plus explaining comment was satisfying enough.